installer/templates/new/web/views/error_helpers.ex

```diff
@@ -26,10 +26,9 @@ defmodule <%= application_module %>.ErrorHelpers do
     #
     #     dngettext "errors", "1 file", "%{count} files", count
     #
-    Gettext.dngettext(<%= application_module %>.Gettext, "errors", msg, msg, opts[:count], opts)
-  end
-
-  def translate_error(msg) do
-    Gettext.dgettext(<%= application_module %>.Gettext, "errors", msg)
+    case Keyword.has_key?(opts, :count) do
```

Comment:
The comment uses dngettext, that's why I think it should apply to only one branch.